frontend/apps/variant-exploration/src/feature/occurrence-table/clinical-association-section.tsx

```diff
@@ -11,26 +11,29 @@ type ClinicalAssociationSectionProps = {
 export default function ClinicalAssociationSection({ data }: ClinicalAssociationSectionProps) {
   const { t } = useI18n();
 
-  const omimCode = useCallback((oc: string[]) => oc.map((ic: string) => <Badge key={ic}>{ic}</Badge>), []);
+  const clinicalAssociationTitle = t('common.noDataAvailable');
+
+  const omimCode = useCallback((oc: string[]) =>
+    oc.map((ic: string) => <Badge key={ic}>{ic}</Badge>),
+    []);
 
   const clinicalAssociationValue = (
     <>
-      {data.omim_conditions?.map(oc => (
+      {data.omim_conditions?.map((oc) => (
```

Comment:
J'ajouterais du spacing:

```suggestion
<div className="space-x-1">
      {data.omim_conditions?.map((oc) => (
        <DetailItem
          title={oc.panel ? oc.panel : clinicalAssociationTitle}
          value={oc.inheritance_code ? omimCode(oc.inheritance_code) : '-'} />
      ))}
    </div>
```